### PR TITLE
Update participants.tsv

### PR DESCRIPTION
**Pull request by renelabounek**
updated sex and age for sub-10062 and sub-10063

### CODE REVIEW

File: participants.tsv [hunk 1]
```diff
@@ -11,4 +11,4 @@ sub-9612	n/a	n/a
 sub-9613	n/a	n/a
 sub-9669	n/a	n/a
 sub-9709	n/a	n/a
-sub-9710	n/a	n/a
\ No newline at end of file
+sub-9710	n/a	n/a
```
Comment (jcohenadad) [inline participants.tsv:14]: @renelabounek can you please undo this change

Comment (renelabounek) [reply participants.tsv:14]: I would love to, but I have no idea how. Or why there is a change? As I have only modified lines 2 and 3. I wil ltry to find it, but I do not see it.

Comment (renelabounek) [reply participants.tsv:14]: @jcohenadad Interenet says, here should be some [Revert button](https://stackoverflow.com/questions/25101983/can-i-revert-commits-directly-on-github), but I do not see it.

Is not easiest if you or @alexfoias will reject my commit and edit sub-10062 and sub-10063 with string "f 26"?
